### ISSUE TITLE
[Bug] Pagination Change To 'Type Links' Not Working

### ISSUE THREAD

**Issue by ievgenklymenko** [laravel/nova-issues#1624]
- Laravel Version: 5.7.28
- Nova Version: 1.3.2
- PHP Version: 7.2.2
- Operating System and Version: macOS Mojave 10.14.5
- Browser type and version: Google Chrome 74.0.3729.157 (Official Build) (64-bit)

### Description:

I'm following this docs to make my pagination more detailed:
https://nova.laravel.com/docs/1.0/resources/#pagination
I have added to my config/nova.php this code:
```
    /*
    |--------------------------------------------------------------------------
    | Nova Settings
    |--------------------------------------------------------------------------
    |
    | Miscellaneous settings.
    |
    */

    'pagination' => 'links',
```

Still, my pagination remains of type "simple".
Tried both `nova:publish` and `nova:publish --force`. Still, after re-caching pagination remains the same.

### Steps To Reproduce:

1. Install Laravel Nova 1.3
2. Add "'pagination' => 'links'" to config/nova.php

**Comment by jnorms** [comment]
The same happens to me in version 2.0.5

**Comment by naifalshaye** [comment]
Same here v1.3.2

**Comment by davidhemphill** [comment]
This feature shipped in 2.0 and is not available on 1.x.

**Comment by jnorms** [comment]
is in the documentation as if it were supported in version 1.x.  But even if it is not supported I can not use it in version 2.0.5.
